tmt/steps/provision/vagrant.py

```diff
@@ -322,7 +480,7 @@ def how_libvirt(self):
         except GeneralError as error:
             self.vf_restore()
             # Not really an error
-            #self.debug(error)
+            self.debug(error)
 
     def how_connect(self):
```

Comment:
Is this one needed? What benefits does it bring to the already implemented `connect` plugin? I see it is also not included in the supported methods.

Reply:
Let me add I'll add it to methods then.

It's an alternative implementation of connect plugin - and it actually works as any other Vagrant plugin, which means it's able to leverage capabilities Vagrant can offer (cross-platform, multi-host, custom network setup, NFS sync, powershell...).

Reply:
Ok, I see, thanks for clarification.